### PR TITLE
IntroScreen&MainScreen

### PR DESCRIPTION
**Pull request by vitalyaliskevych**
I created IntoScreen& MainScreen [https://trello.com/c/aIBOnrSH](url) [https://trello.com/c/mGp6X5rP](url)
Screenshot 2022-12-04 at 13 24 34
Screenshot 2022-12-04 at 13 24 45

### CODE REVIEW

**Comment by vasyl-nadtochii** [comment]
Divide your PR description into sections:

- **What's new**: short description of what has been changed
- **Issue**: link to the ticket(-s)
- **Screenshot/Video**
- **Notes**: optional, place here any additional information

Also, please, don't forget to setup reviewers and add label (for new feature use `enhancement`, for bug fix `bug` etc...)

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+//  Crypto
+//
+//  Created by Developer on 17.11.2022.
+//
+
+import SwiftUI
+
+struct MainView: View {
+    init() {
+        UINavigationBar.appearance().backgroundColor = UIColor(Color.bottomButtonColor)
```
Comment (vasyl-nadtochii) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:12]: we've decided not to make navigation bar coloured, so remove this code. Also, it's not good solution, cuz it is affecting navigation bars across whole application, making potentially unwilled consequences. Refer to hifi (high fidelity design in Figma)

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    }
+    var body: some View {
+        NavigationView {
+            ZStack {
+                VStack {
+                    VStack {
+
```
Comment (vasyl-nadtochii) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:19]: remove this spacing and avoid having extra spacings across whole project

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    }
+    
+}
+
+extension MainView {
+    
+    var topCoin: some View {
+        VStack {
+            VStack(spacing: 10) {
+                //            btcBlock
```
Comment (vasyl-nadtochii) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:49]: commented code should be removed or marked with `TODO` with appropriate comment if it's needed for the future tasks

Example:
```
// TODO: temporary code - will be removed once request is implemented
/* your commented code */
```

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+                ethBlock
+                btcBlock
+                ltcBlock
+                ethBlock
+            }
+            Spacer()
+            stgBlock
+        }
+    }
+    var ltcBlock: some View {
```
Comment (vasyl-nadtochii) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:66]: this is wrong approach for the reusable views.

List cells should be reusable `func` that returns `some View`, in which we just pass `image`, `name` and `price` parameters. And then in `ForEach` loop you just go through some array and call this func. 

Move this logic to the function in this `extension` and create some dummy set of data

**UPD**: I can see that you already have such func, but still please, remove `ethBlock`, `btcBlock`, `ltcBlock`

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    
+    struct MainView_Previews: PreviewProvider {
+        static var previews: some View {
+            MainView()
+        }
+    }
+    
+}
+
+extension MainView {
```
Comment (vasyl-nadtochii) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:44]: make it `private`

File: Crypto/Flows/Screens/SettingsScreen/SettingsView.swift [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+//
+//  Created by Developer on 01.12.2022.
+//
+
+import SwiftUI
+
+struct SettingsView: View {
+    var body: some View {
+        VStack {
+
```
Comment (vasyl-nadtochii) [inline Crypto/Flows/Screens/SettingsScreen/SettingsView.swift:13]: does it even compile? Place `EmptyView` here.

But this screen is out of the scope of this task, so please try not to add it before actual task

File: Crypto/Flows/Screens/MainScreen/MainView.swift [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+    }
+    var btcBlock: some View {
+        VStack {
+            createButton(imagName: Image.btcIcn, text: "Bitcoin", number: "21,188.12")
+        }
+        .background(Color.bottomButtonColor)
+        .cornerRadius(15)
+    }
+    
+    var stgBlock: some View {
```
Comment (vasyl-nadtochii) [inline Crypto/Flows/Screens/MainScreen/MainView.swift:90]: try to use full names - `settingsBlock` in this case

check across the project this and change

**Review by vasyl-nadtochii — COMMENTED** [review]
Please, fix next comments

File: Crypto/Flows/Screens/MainScreen/MainView.swift (hunk 1 above)
Comment (vitalyaliskevych) [reply Crypto/Flows/Screens/MainScreen/MainView.swift:12]: Done

**Pull request merged**